open-scd.spec.ts

```diff
@@ -0,0 +1,152 @@
+import { expect } from '@open-wc/testing';
+
+import './open-scd.js';
+import type { OpenSCD } from './open-scd.js';
+
+import { newEditEvent, newOpenEvent } from './foundation.js';
+
+function isOscdPlugin(tag: string): boolean {
+  return tag.toLocaleLowerCase().startsWith('oscd-');
```

Comment:
Actually, our UI components will also start with `oscd-` (e.g. `oscd-tree-explorer`), so it might be a bit safer to look for startsWith `oscd-p` and check the `length`. I don't expect this to cause any actual trouble, just being thorough here.

Reply:
Look at that again what learned :D. I have not even noticed that the loaded plugins custom tag name by design start with `oscd-p`